adafruit_led_animation/animation/__init__.py

```diff
@@ -201,11 +201,11 @@ def speed(self):
         """
         The animation speed in fractional seconds.
         """
-        return self._speed_ns / NANOS_PER_SECOND
+        return self._speed_ns / MS_PER_SECOND
```

Comment:
Change `self._speed_ns` to `self._speed_ms`? Cosmetic, but could help prevent future confusion.